learn/rhpds/optaweb-vehicle-routing.adoc

```diff
@@ -3,56 +3,14 @@
 :awestruct-layout: normalBase
 :awestruct-priority: 0.1
 :showtitle:
+:demo-catalog-item: DM7 OptaWeb VRP Demo
```

Comment:
Are you sure we want to hardcode the version "7" here? I 'd avoid mentioning version numbers.

Also, I think we better replace "VRP" with "Vehicle Routing", as not everyone will be familiar with the vrp acronym. Applies to all optaweb-vehicle-routing documentation and mentioning btw.

But of course, all this is already out in the catalog, so we can't change it now. So let's merge this and work towards it, wdyt?